### DISCUSSION TITLE
What are some of the best programming languages in 2023?

### DISCUSSION THREAD

**Discussion by The-Leo** [streetprophet90/Synergy-Codes#12] [Q&A] [answered]
What are some of the best programming languages in 2023?

**Comment by deedadey-dd** [comment, marked as answer]
Many people will rather shy away from mentioning any specific language but I won't shudder to rank them as follows
* Python
* Javascript, HTML, CSS
* C
With these I believe you can be whatever you want to be.

**Comment by The-Leo** [comment]
I'll go for Python and C++
